### ISSUE TITLE
Move logos from yunohost/apps to apps repositories

### ISSUE THREAD

**Issue by Salamandar** [YunoHost/issues#2249]
It would make more sense for logos to be in apps' repositories:
* maintainers can add them without too much effort
* they really are part of the package
* logos can then be visible in the README

**Comment by zamentur** [comment]
However, it could be more difficult to have graphical consistency. A lot of logos have been retourched by ericg in order to get a consistency and an nice display of the catalog. I say me that changing logo in each repo could be a little more work than doing it in a single repo...

**Comment by Thovi98** [comment]
Yeah I agree, and the logo has to be uploaded, it is just a matter of which repo at the end…
And about

> logos can then be visible in the README

It's already the case now
